src/components/application_manager/include/application_manager/application_data_impl.h

```diff
@@ -263,6 +267,10 @@ class DynamicApplicationDataImpl : public virtual Application {
    */
   inline bool is_reset_global_properties_active() const;
 
+  virtual void set_choice_set_allow_mode(const uint32_t choice_set_id,
+                                         const bool is_allowed);
```

Comment:
@IGapchuk please use OVERRIDE or FINAL macros from `macro.h` and remove virtual

Reply:
@AKalinich-Luxoft Fixed in 466447c82ad8882ea67b47f96fcc10788db6351f.